polling_stations/apps/addressbase/management/commands/create_uprn_council_lookup.py

```diff
@@ -0,0 +1,76 @@
+from django.core.management.base import BaseCommand
+from django.db import connection
+from pathlib import Path
+
+
+class Command(BaseCommand):
+    """
+    This creates a lookup csv of uprn and council GSS codes.
+    This can then be imported using 'import_uprn_council_lookup'.
+    """
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "-d", "--destination", help="Path to write csv to", default=None
+        )
+
+    def handle(self, *args, **kwargs):
+
+        self.cursor = connection.cursor()  # Does this need to specify DEFAULT?
```

Comment:
Nope - it uses by default by default (turns out the clue's in the name :smile:)
We only need to specify the connection if we want the logger DB